core/src/cnf/mod.rs

```diff
@@ -42,3 +51,32 @@ pub static INSECURE_FORWARD_SCOPE_ERRORS: Lazy<bool> = Lazy::new(|| {
 		.and_then(|s| s.parse::<bool>().ok())
 		.unwrap_or(false)
 });
+
+/// Specifies the path of the temporary directory used by SurrealDB.
+/// If not specified, SurrealDB attempts to make a temporary directory inside `env::temp_dir()`.
```

Comment:
On many Linux distributions, the /tmp directory is often mounted as a RAM disk, known as tmpfs. This means that it is a filesystem residing in the system's memory (RAM) rather than on a physical disk or storage device. The temporary directory is not a different filesystem on the Mac, so one might see different results there. Just wanted to share this as we are defaulting to `env::temp_dir` here

Reply:
That's a very valid point.
Then currently the directory used by file-based datastore (RocksDB, Speedb, Surreal-KV)  is fully managed by the KV store itself. So I didn't want to interfere there. And for remote-based datastore (FDB, Tikv) there is no designed local file system location.
With this new environment variable and document it, users can easily set up an alternative location. And with the default value, there is no breaking changes in the way we start the server.

Reply:
Noted! Thank you